collector/src/compile/execute/rustc.rs

```diff
@@ -193,9 +193,9 @@ fn checkout(artifact: &ArtifactId) -> anyhow::Result<()> {
             .current_dir("rust")
             .arg("fetch")
             .arg("origin")
-            .arg("master")
+            .arg("HEAD")
```

Comment:
Does this update the reference that bootstrap is reading? My memory is hazy (and I'm not sure how to test) but I think I've seen git not update e.g. origin/master for this kind of command...